### DISCUSSION TITLE
Site unusable from CORS errors right now

### DISCUSSION THREAD

**Discussion by kyleyakamoto** [WalletConnect/walletconnect-monorepo#3540] [Developer Support]
Right now I can't add any URLs to the project I just crated. I see a toast saying "Failed to Fetch" and the following errors in the console:

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/105392865/dc6f855a-5b9e-4e3b-93e2-d4a64222e1bb)

**Comment by boidushya** [comment]
Could you elaborate on this issue? A reproducible example or code snippet of where you're trying to add URLs?
